FewCore/Element.swift

```diff
@@ -116,7 +118,9 @@ public class Element {
 				compareAndSetAlpha(view, alpha)
 			}
 			
-			realizedSelf.element = self
+			if old.frame.size != frame.size {
```

Comment:
Hm that's no good. How can we achieve this effect without all the false positives?